documentation/src/test/java/example/ExampleDependsOnAnnotationTestCase.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package example;
+
+// tag::user_guide[]
+
+import org.junit.jupiter.api.DependsOn;
+import org.junit.jupiter.api.MethodOrderer.DependsOnAnnotation;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(DependsOnAnnotation.class)
+public class ExampleDependsOnAnnotationTestCase {
+	@Test
+	void d() {
+		System.out.println("Test 1");
+	}
+
+	@Test
+	@DependsOn(value = "d")
```

Comment:
The reason we switched from `methods` to `value` is so that developers do not have to type `value =`.

So, please change all occurrences of `@DependsOn(value = "X")` to `@DependsOn("X")`. 😉

Reply:
Done and the new test is also automated.